micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jersey/server/JerseyContext.java

```diff
@@ -36,7 +36,7 @@ public class JerseyContext extends RequestReplyReceiverContext<ContainerRequest,
     public JerseyContext(RequestEvent requestEvent) {
         super((carrier, key) -> {
             List<String> requestHeader = carrier.getRequestHeader(key);
-            if (requestHeader.isEmpty()) {
+            if (requestHeader == null || requestHeader.isEmpty()) {
```

Comment:
A NPE was being thrown when trying to use the Getter on a key that wasn't in the headers. The above `getRequestHeader` returns null for an absent key.